Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1070,11 +1070,12 @@ def compile(self, more, haveInstructions, glyfTable):
 				data = data + struct.pack(">HH", self.firstPt, self.secondPt)
 				flags = flags | ARG_1_AND_2_ARE_WORDS
 		else:
+			x, y = self.roundOffset()
```

Comment:
I just thought may be we want to keep the original coordinates for other operations e.g. when saving to XML, and having the saved file be different whether one saved before or after compiling the table seems unpredictable IMO.